pkg/controller/context.go

```diff
@@ -31,6 +32,7 @@ const (
 	contextKeySession       = contextKey("session")
 	contextKeyTemplate      = contextKey("template")
 	contextKeyUser          = contextKey("user")
+	contextKeyFBUser        = contextKey("fbUser")
```

Comment:
I'd prefer if we spelled out firebase, especially given fb could mean facebook...